src/Http/Router.php

```diff
@@ -63,7 +63,7 @@ private function get_current_route() {
 		// requested uri relative to WP install
 		$request_uri = str_replace( $installed_dir, '', $_SERVER['REQUEST_URI'] );
 
-		$uri   = sanitize_text_field( wp_unslash( $request_uri ) );
+		$uri   = sanitize_url( wp_unslash( $request_uri ) );
```

Comment:
Nice catch! I looked at its PHPDOC and it says it has been un-deprecated now?

```
 * @since 2.3.1
 * @since 2.8.0 Deprecated in favor of esc_url_raw().
 * @since 5.9.0 Restored (un-deprecated).
```

I will go ahead and still change it anyway.

Reply:
oh wow, bug in phpcs then.